### PR TITLE
[GL- 1599] Build and use SAMTOOLS + PICARD + BWA-specific docker images instead of GOTC

### PR DESCRIPTION
**Pull request by cheyenne-gold**
This new image should meet the needs of Alignment.wdl, follow Docker best practices and be publicly available.

### CODE REVIEW

**Comment by cheyenne-gold** [comment]
retest this please

File: dockers/broad/samtools_picard_bwa/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        \
+        # BWA
+        wget ${BWA_URL} -O bwakit-${BWA_VERSION}.tar.bz2; \
+        tar xf bwakit-${BWA_VERSION}.tar.bz2;  \
+        rm bwakit-${BWA_VERSION}.tar.bz2; \
+        mv bwa.kit/bwa bwa; \
+        rm -rf bwa.kit; \
+        \
+        # PICARD
+        wget ${PICARD_URL};
```
Comment (wjdingman1) [inline dockers/broad/samtools_picard_bwa/Dockerfile:34]:
```suggestion
# Install BWA
        mkdir temp; \
        cd temp; \
        \
        wget ${BWA_URL} -O bwakit-${BWA_VERSION}.tar.bz2; \
        tar xf bwakit-${BWA_VERSION}.tar.bz2;  \
        mv bwa.kit/bwa ../bwa; \
        \
        cd ../; \
        rm -r temp \
        ; \
# Install picard
        wget ${PICARD_URL};
```

File: dockers/broad/samtools_picard_bwa/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        PICARD_PUBLIC_VERSION=${PICARD_PUBLIC_VERSION}
+
+WORKDIR /usr/gitc
+
+# Install dependencies
+RUN set -eux; \
+        apk add --no-cache \
+        gcompat \
+        libc6-compat \
+        openjdk8 \
```
Comment (wjdingman1) [inline dockers/broad/samtools_picard_bwa/Dockerfile:21]:
```suggestion
            gcompat \
            libc6-compat \
            openjdk8 \
```

**Review by wjdingman1 — APPROVED** [review]
Looks good except for small formatting changes!

**Pull request merged**